cli/image-scanning-by-using-the-roxctl-cli.adoc

```diff
@@ -0,0 +1,29 @@
+[id="image-scanning-by-using-the-roxctl-cli"]
+:_content-type: ASSEMBLY
+= Image scanning by using the roxctl CLI
+include::modules/common-attributes.adoc[]
+:context: image-scanning
+
+toc::[]
+
+[role="_abstract"]
+{ocp} provides you with an integrated image registry for storing images, called the registry. By default, the registry is only accessible within the cluster where it's deployed, and each cluster can have its own registry.
+
+The URI of an image within the registry contains information about the {ocp} project and namespace from which you can access the image. The typical URI format is `_<registry_ip>:<port>_/_<project>_/_<image>_`.
+
+For example, an image such as `image-registry.openshift-image-registry.svc:5000/helloworld/image` is only accessible from the `helloworld` project, where `image-registry.openshift-image-registry.svc:5000` is the default hostname.
+
+To scan images from the registry using `roxctl` CLI, Jenkins or API, you need to ensure that each of these systems supports delegating scans to specific secured clusters. It is important that you extract the project and namespace from the image URI, and you should use project and namespace specific credentials to retrieve or scan the image.
```

Comment:
The `roxctl` CLI can be used to scan images from any registry, it isn't limited to just the OCP integrated registry. 

Recommend either removing this entire section, or changing it to provide general information about scanning images with roxctl (not specific to the OCP integrated registry).